keps/0000-kep-template.md

```diff
@@ -0,0 +1,162 @@
+---
+kep-number: 0
+title: My First KEP
+authors:
+  - "@janedoe"
+reviewers:
```

Comment:
Reviewers + Approvals feels unnecessary for our current scope. We just have a top level OWNERS.